Sources/Fetchers/SdJwtVcIssuerMetaDataFetcher.swift

```diff
@@ -15,11 +15,11 @@
  */
 import Foundation
 import SwiftyJSON
-import JSONWebKey
+@preconcurrency import JSONWebKey
```

Comment:
Hey @dtsiflit . I think version 4.0.1 already has swift 6 support ;) so this would not be needed.

Reply:
Thanks so much @goncalo-frade-iohk :) I've updated the PR to reflect this.